### DISCUSSION TITLE
Remove empty grid cells from showOutsideDays

### DISCUSSION THREAD

**Discussion by jvfalco1** [gpbl/react-day-picker#2060] [Support]
Is it possible to remove the empty cells from ```showOutsideDays={false}```?

I want to show my calendar always starting from the first grid cell example:

![image](https://github.com/gpbl/react-day-picker/assets/39075106/4dbffe0f-87ad-44b1-9933-3dd2792d23ee)

**Comment by gpbl** [comment]
@jvfalco1 not sure what your are trying to accomplish here. Would you mind to explain? Thanks!
